pkg/kubelet/network/hostport/hostport.go

```diff
@@ -172,35 +172,23 @@ func hostportChainName(cp api.ContainerPort, podFullName string) utiliptables.Ch
 
 // OpenPodHostportsAndSync opens hostports for a new pod, gathers all hostports on
 // node, sets up iptables rules enable them. And finally clean up stale hostports
-func (h *handler) OpenPodHostportsAndSync(newPod *RunningPod, natInterfaceName string, runningPods []*RunningPod) error {
+func (h *handler) OpenPodHostportsAndSync(newPod *api.Pod, natInterfaceName string, activePods []*ActivePod) error {
```

Comment:
Should mention in the function doc here that newPod must also be present in 'activePods', and I think this function should enforce that too.  Otherwise it's error-prone.